### PR TITLE
Amendments for uni

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Resolves a change in uni-for-after branch https://github.com/bbc/simorgh/pull/640

In uni we removed sw cached externals, this makes public folder actually public and caches them with sw. Also changes start url in manifest to a page this application serves.

**Test Description**
Spin up locally with `npm run build` & then `npm run start`.
Visit `http://localhost:7080/news/articles/c0000000025o` in Chrome
Open dev tools, go to network tab.
Upon refreshing page, ensure font files are loaded from service worker. This should appear as:

screen shot 2018-10-04 at 14 11 32

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by bcmn** [comment]
Quick observation, can't claim to have checked everything -- After.js is no longer present in the `package.json`, but we have a few remaining references to it in documentation:
[Lines 46 through 50 of the Article container](https://github.com/bbc/simorgh/blob/amendments-for-uni/src/app/containers/Article/index.jsx#L47)
[Line 9 of the README](https://github.com/bbc/simorgh/blob/amendments-for-uni/README.md#L9)

File: razzle.config.js [hunk 1]
```diff
@@ -42,6 +42,17 @@ module.exports = {
             // externals means files not generated by webpack
             // an alternative would be html-webpack-plugin
             caches: 'all',
+            externals: [
+              '/public/manifest.json',
+              '/public/images/icons/icon-72x72.png',
```
Comment (jamesbhobbs) [inline razzle.config.js]: do we need these images here?

**Comment by jamesbhobbs** [comment]
@bcmn great point can you open another mini-PR to fix documentation separately please

File: razzle.config.js (hunk 1 above)
Comment (jtart) [reply razzle.config.js]: I think these only need to be here if the application makes a network request for them. I do not think they need to be here if they are here for PWA, manifest, reasons.

File: razzle.config.js [hunk 2]
```diff
@@ -42,6 +42,17 @@ module.exports = {
             // externals means files not generated by webpack
             // an alternative would be html-webpack-plugin
             caches: 'all',
+            externals: [
```
Comment (jtart) [inline razzle.config.js:45]: Do we want to serve fonts here?

File: src/server/index.jsx [hunk 3]
```diff
@@ -48,6 +48,7 @@ const server = express();
 server
   .disable('x-powered-by')
   .use('/data', express.static(dataFolderToRender))
+  .use('/public', express.static('public'))
```
Comment (jtart) [inline src/server/index.jsx]: We serve public files in the line below using the razzle public directory, which moves `public` files to `build` for prod/serves from `public` for dev. This line will just serve everything from `public`, we should just use the line below.

File: public/manifest.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "Scope": "/news/",
   "background_color": "#b80000",
   "display": "standalone",
-  "start_url": "/news/",
+  "start_url": "/news/articles/c0000000025o",
```
Comment (jtart) [inline public/manifest.json:7]: Should this be a URL for a specific article?

Would usually be something like an index link here.

**Review by jtart — CHANGES_REQUESTED** [review]

**Comment by bcmn** [comment]
> @bcmn great point can you open another mini-PR to fix documentation separately please

Done, see #712.

File: public/manifest.json (hunk 4 above)
Comment (jamesbhobbs) [reply public/manifest.json:7]: No, but we don't have a choice atm as we serve no other pages. We will add one here https://github.com/BBC-News/simorgh/issues/755

File: razzle.config.js (hunk 2 above)
Comment (jamesbhobbs) [reply razzle.config.js:45]: Updated, I also revised how we inject fonts, so we only inject fonts we actually use. I did this here not separately as it clarifies why only some are cached.

**Review by pjlee11 — DISMISSED** [review]

File: src/app/lib/globalStyles.js [hunk 5]
```diff
@@ -30,17 +23,20 @@ injectGlobal`
...
-    font-family: ReithSansNewsMedium;
+    font-family: ReithSerifNewsMedium;
     font-style: normal;
     font-weight: 600;
-    src: url('https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Md.woff2') format('woff2'), url('https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Md.woff') format('woff');
+    src: url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff2') format('woff2'), url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff') format('woff');
   }
+`;
+
+/* Unused fonts
```
Comment (jtart) [inline src/app/lib/globalStyles.js:33]: Can we add a comment to be sure to add the font to the `externals` array so we don't forget?

**Review by jtart — DISMISSED** [review]

**Review by bcmn — DISMISSED** [review]

**Review by bcmn — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Pull request merged**